### PR TITLE
Add machine coin rates screen

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

File: lib/plugins.js [hunk 1]
```diff
@@ -231,6 +231,7 @@ function plugins (settings, deviceId) {
     const localeConfig = configManager.getLocale(deviceId, settings.config)
     const fiatCode = localeConfig.fiatCurrency
     const cryptoCodes = localeConfig.cryptoCurrencies
+    const machineScreenOpts = configManager.getAllMachineScreenOpts(settings.config)
```
Comment (siiky) [inline lib/plugins.js:234]: `getMachineScreenOpts(screenName, config)` but only the config is given here (and in the wrong order).

File: lib/graphql/resolvers.js [hunk 2]
```diff
@@ -99,6 +106,7 @@ const staticConfig = ({ currentConfigVersion, deviceId, deviceName, pq, settings
     configManager.getLocale(deviceId, settings.config),
     configManager.getOperatorInfo(settings.config),
     configManager.getReceipt(settings.config),
+    configManager.getAllMachineScreenOpts(settings.config),
```
Comment (siiky) [inline lib/graphql/resolvers.js:114]: `addSmthInfo()` checks if the given `smth` has an `active` value and is truthy, but looking at `getAllMachineScreenOpts()` I don't think there's any.

**Review by siiky — CHANGES_REQUESTED** [review]

File: lib/graphql/resolvers.js (hunk 2 above)
Comment (chaotixkilla) [reply lib/graphql/resolvers.js:114]: fixed

File: lib/graphql/resolvers.js [hunk 3]
```diff
@@ -131,7 +145,8 @@ const staticConfig = ({ currentConfigVersion, deviceId, deviceName, pq, settings
         }),
         _.update('triggersAutomation', _.mapValues(_.eq('Automatic'))),
         addOperatorInfo(operatorInfo),
-        addReceiptInfo(receiptInfo)
+        addReceiptInfo(receiptInfo),
+        _.update('screenOptions', addMachineScreenOpts(machineScreenOpts))
```
Comment (siiky) [inline lib/graphql/resolvers.js]: Isn't this `_.update()` unnecessary here? It's already in `addMachineScreenOpts()`.

**Review by siiky — CHANGES_REQUESTED** [review]

**Review by siiky — APPROVED** [review]

**Pull request merged**